zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/NotebookImportDeserializer.java

```diff
@@ -35,7 +35,7 @@
 public class NotebookImportDeserializer implements JsonDeserializer<Date> {
   private static final String[] DATE_FORMATS = new String[] {
     "yyyy-MM-dd'T'HH:mm:ssZ",
-    "MMM dd, yyyy HH:mm:ss"
+    "MMM d, yyyy h:mm:ss a"
```

Comment:
Thanks for the contribution.

Because of deserialize() tries all formats in `DATE_FORMATS` sequentially, i think we can keep both formats. "MMM d, yyyy h:mm:ss a" and then "MMM dd, yyyy HH:mm:ss".